src/android/InAppBrowser.java

```diff
@@ -633,6 +650,12 @@ private void goForward() {
         }
     }
 
+    private void doReload(){
+        if (this.inAppWebView != null) {
```

Comment:
Is this check actually safe? Can `this.inAppWebView` and `inAppWebView` be different?

Reply:
Good catch! It should have been `this.inAppWebView.reload()`